app/views/list/_request_row_template.slim

```diff
@@ -0,0 +1,95 @@
+tr.request
+  td.request-checkbox
+    input[type="checkbox"]
+  td.request-edit
+    a[href="#"]
+      | Edit
+  td.request-name
+    a[href="#"]
+      | Rubik's Cube Game
```

Comment:
This should be modified to display the data for an actual request passed to it, then you can just generate a test page using a test model.